pkg/controller/vpcpeering/peering.go

```diff
@@ -571,19 +571,20 @@ func (e *external) deleteRoute(ctx context.Context, client peering.EC2Client, na
 		return errors.Wrap(err, "describe RouteTables")
 	}
 
-	e.log.WithValues("VpcPeering", name).Debug("Describe RouteTables for deleting", "result", routeTablesRes.String())
+	log.WithValues("VpcPeering", name).Debug("Describe RouteTables for deleting", "result", routeTablesRes.String())
 	for _, rt := range routeTablesRes.RouteTables {
 		for _, r := range rt.Routes {
 			if r.VpcPeeringConnectionId != nil && pcx != nil && *r.VpcPeeringConnectionId == *pcx {
-				_, err := e.client.DeleteRouteRequest(&ec2.DeleteRouteInput{
+				_, err := client.DeleteRouteRequest(&ec2.DeleteRouteInput{
 					DestinationCidrBlock: aws.String(peerCIDR),
-
-					RouteTableId: rt.RouteTableId,
+					RouteTableId:         rt.RouteTableId,
 				}).Send(ctx)
 				if err != nil {
-					return errors.Wrap(err, "delete Route")
+					if !strings.Contains(err.Error(), "InvalidRoute.NotFound") {
```

Comment:
swallow this error ？

Reply:
This is to ensure that the execution of the program is idempotent